documentation/release_notes.markdown

```diff
@@ -10,11 +10,60 @@ canonical: "/puppetserver/latest/release_notes.html"
 [configuration directory]: /puppet/latest/reference/dirs_confdir.html
 [code directory]: /puppet/latest/reference/dirs_codedir.html
 [cache directory]: /puppet/latest/reference/dirs_vardir.html
+[`trapperkeeper-authorization`]: https://github.com/puppetlabs/trapperkeeper-authorization
```

Comment:
No.  I think this might have been an error from the release notes writeup.  Nothing references this link in the docs.  /CC @gguillotte.